### DISCUSSION TITLE
Old version (1.1.5) showing in-editor

### DISCUSSION THREAD

**Discussion by DavidOliver** [primer/github-vscode-theme#313] [General]
I'm trying to install the latest version in order to get the darker dark theme, but VSCodium (1.55.2, Linux) is only showing me version 1.1.5.

Thanks.

**Comment by DavidOliver** [comment]
I've since discovered this is because VSCodium uses https://open-vsx.org/ by default: https://github.com/VSCodium/vscodium/blob/master/DOCS.md#extensions-marketplace

In case the owner of this theme is interested in publishing updates to Open VSX: https://github.com/eclipse/openvsx/wiki/Publishing-Extensions

**Comment by dominicegginton** [comment]
@DavidOliver I use Code OSS on Linux based OS and have found this [this](https://stackoverflow.com/questions/64463768/cant-find-certain-extensions-in-code-ossopen-source-variant-of-visual-studio-c) workaround to install and update exertions and themes that are published on the visual studio code marketplace but not on open-vsx. Once you switch to using the visual studio code marketplace you will be able to update this theme to the latest version. Then switch back to using the open-vsx marketplace if you prefer to use that.

**Comment by DavidOliver** [comment]
@dominicegginton, thanks - I now have the updated theme. :)

Using Microsoft's marketplace in VSCodium is also documented here: https://github.com/VSCodium/vscodium/blob/master/DOCS.md#how-to-use-the-vs-code-marketplace

**Comment by sau226** [comment]
I went and filed a PR in a repo that can auto-publish the extension to the openvsx platform (linked above)
